condominium/demo/project_project.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="project_project_4" model="project.project">
+        <field name="analytic_account_id" ref="account_analytic_account_4"/>
```

Comment:
Why did you remove the analytic account of other (demo) projects?

Reply:
Sorry it was removed by mistake I have added it back